apps/manager_app/onboarding/onboarding_host_interface.c

```diff
@@ -59,9 +59,10 @@
 /*****************************************************************************
  * INCLUDES
  *****************************************************************************/
-#include "host_interface.h"
+#include "onboarding_host_interface.h"
```

Comment:
Note: The function definition is out of scope of this PR so commenting here

Rename `host_interface` function to `onboarding_host_interface` as well.

Reply:
updated